### PR TITLE
[ISSUE #5776] Remove duplicate empty string checks

### PR DESCRIPTION
**Pull request by yx9o**
For #5776 , remove repeated empty string checks for `topic` and `group`, and add unit tests.

org.apache.rocketmq.client.Validators#checkTopic：
image
org.apache.rocketmq.client.Validators#checkGroup：
image

### CODE REVIEW

**Comment by mxsm** [comment]
Hey @yx9o  Thank you for your contribution, But you can create issue that describes the problem before you submit a pr. then you can submit a pr  and link this pr to issue. how to link you can see this document:https://docs.github.com/zh/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

**Comment by yx9o** [comment]
> Hey @yx9o Thank you for your contribution, But you can create issue that describes the problem before you submit a pr. then you can submit a pr and link this pr to issue. how to link you can see this document:https://docs.github.com/zh/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

Thanks for your reply, adjusted.

File: proxy/src/main/java/org/apache/rocketmq/proxy/grpc/v2/common/GrpcValidator.java [hunk 1]
```diff
@@ -106,16 +100,17 @@ public void validateInvisibleTime(long invisibleTime, long minInvisibleTime) {
...
+            return;
+        }
+        if (StringUtils.isBlank(tag)) {
+            throw new GrpcProxyException(Code.ILLEGAL_MESSAGE_TAG, "tag cannot be the char sequence of whitespace");
+        }
+        if (tag.contains("|")) {
+            throw new GrpcProxyException(Code.ILLEGAL_MESSAGE_TAG, "tag cannot contain '|'");
+        }
+        if (containControlCharacter(tag)) {
+            throw new GrpcProxyException(Code.ILLEGAL_MESSAGE_TAG, "tag cannot contain control character");
```
Comment (caigy) [inline proxy/src/main/java/org/apache/rocketmq/proxy/grpc/v2/common/GrpcValidator.java:113]: Is this modification related to this issue? If not, you'd better move it to another issue.

**Review by caigy — CHANGES_REQUESTED** [review]

File: proxy/src/main/java/org/apache/rocketmq/proxy/grpc/v2/common/GrpcValidator.java (hunk 1 above)
Comment (yx9o) [reply proxy/src/main/java/org/apache/rocketmq/proxy/grpc/v2/common/GrpcValidator.java]: Ok, thanks for your review, I modified it and I will add it to a new issue.

**Review by mxsm — APPROVED** [review]
LGTM~

**Review by lizhanhui — APPROVED** [review]

**Review by caigy — APPROVED** [review]

**Review by hzh0425 — APPROVED** [review]
